README.md

```diff
@@ -55,45 +56,46 @@ As of now, these are the supported resources:
      | InventoryEntry  | `sku`  |
      | State  | `key`  |
      | TaxCategory  | `key`  |
- 
+     | CustomObject  | `container` AND `key`  |
+
  - Set the following environment variables before running the application
    ```bash
    export SOURCE_PROJECT_KEY = "source-project-key"
    export SOURCE_CLIENT_ID = "sourceClientId"
    export SOURCE_CLIENT_SECRET = "sourceClientSecret"
    export SOURCE_AUTH_URL = "https://auth.eu-central-1.aws.commercetools.com/" #optional parameter
    export SOURCE_API_URL = "https://api.eu-central-1.aws.commercetools.com/" #optional parameter
-   
+
    export TARGET_PROJECT_KEY = "target-project-key"
    export TARGET_CLIENT_ID = "targetClientId"
    export TARGET_CLIENT_SECRET = "targetClientSecret"
    export TARGET_AUTH_URL = "https://auth.eu-central-1.aws.commercetools.com/" #optional parameter
    export TARGET_API_URL = "https://api.eu-central-1.aws.commercetools.com/" #optional parameter
    ```
    Note: For *_AUTH_URL and *_API_URL parameter values,
-    you can use different [authentication endpoints](https://docs.commercetools.com/http-api-authorization#requesting-an-access-token-using-commercetools-oauth2-server) and [API endpoints](https://docs.commercetools.com/http-api#hosts). 
+    you can use different [authentication endpoints](https://docs.commercetools.com/http-api-authorization#requesting-an-access-token-using-commercetools-oauth2-server) and [API endpoints](https://docs.commercetools.com/http-api#hosts).
 
 ### Usage
 
    ```bash
    usage: commercetools-project-sync
     -h,--help               Print help information.
-    -s,--sync <arg>         Choose one of the following modules to run: "types", "productTypes", "states", 
-                            "taxCategories", "categories", "cartDiscounts", "products", "inventoryEntries" or "all" 
+    -s,--sync <arg>         Choose one of the following modules to run: "types", "productTypes", "states",
```

Comment:
customObjects is missing.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/173/commits/f1620b24de401c8d5797f73e4dcd5f33256765d8